ccmbioinfo/lfs
# CARDS-2323: Embedded view of subject charts and forms, without navigation elements

To test:
- start in proms mode
- create a new Visit Information; if the data forms are not automatically created, also create some data forms
- fill in the data forms
- open the visit subject chart
- in the URL, replace `/content.html` with `/embedded.html`, e.g. `http://localhost:8080/embedded.html/Subjects/a10119c8-a9a7-4950-ba7c-48d5ca434dbf/da0201a6-680a-4a5d-8a3f-d59a880e5ae4`
- create a new user, log in with that user and open the embedded view -> error displayed
- as admin, add the user to the Trusted Users group
- as the new user, open again the embedded view -> this time it should display
- repeat these steps for prems mode (except for the Trusted Users part, since prems doesn't have trusted users)
